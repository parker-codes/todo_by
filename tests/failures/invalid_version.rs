use todo_by::todo_by_version;

fn main() {
    todo_by_version!("2024/01/01");
}
