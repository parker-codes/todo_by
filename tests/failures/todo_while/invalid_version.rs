use todo_by::todo_while;

fn main() {
    todo_while!("2024/01/01");
}
