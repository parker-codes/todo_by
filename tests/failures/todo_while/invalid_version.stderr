error: proc macro panicked
 --> tests/failures/todo_while/invalid_version.rs:4:5
  |
4 |     todo_while!("2024/01/01");
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: Not a valid semver requirement: Error("expected comma after major version number, found '/'")
